packages/core/src/login/webview/vue/login.vue

```diff
@@ -71,20 +71,20 @@
                 />
             </svg>
         </div>
+        <div style="margin-bottom: 12px"></div>
         <template v-if="stage === 'START'">
-            <div>
-                <button class="back-button" v-if="app === 'TOOLKIT'" @click="handleBackButtonClick">
-                    <svg width="13" height="11" viewBox="0 0 13 11" fill="none" xmlns="http://www.w3.org/2000/svg">
-                        <path
-                            d="M4.98667 0.0933332L5.73333 0.786666L1.57333 4.94667H12.0267V5.96H1.57333L5.73333 10.0667L4.98667 10.8133L0.0266666 5.8V5.10667L4.98667 0.0933332Z"
-                            fill="#21A2FF"
-                        />
-                    </svg>
-                </button>
-            </div>
+            <button class="back-button" v-if="app === 'TOOLKIT'" @click="handleBackButtonClick">
+                <svg width="13" height="11" viewBox="0 0 13 11" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path
+                        d="M4.98667 0.0933332L5.73333 0.786666L1.57333 4.94667H12.0267V5.96H1.57333L5.73333 10.0667L4.98667 10.8133L0.0266666 5.8V5.10667L4.98667 0.0933332Z"
+                        fill="#21A2FF"
+                    />
+                </svg>
+            </button>
             <div class="auth-container-section">
                 <div class="existing-logins" v-if="existingLogins.length > 0">
                     <div class="header">Connect with an existing account:</div>
+                    <div style="margin-bottom: 12px"></div>
```

Comment:
Make a class instead named something like `bottomMargin` and set this class to the element you care about. Then set the margin in that class and if you need to update, it will apply to all, like a variable

Reply:
done